variables.tf

```diff
@@ -2,8 +2,95 @@
+
+variable "name" {
+  type        = string
+  description = "The name to give the IBM Event Notification instance created by this module."
+}
+
+variable "plan" {
+  type        = string
+  description = "Plan for the event notification instance : lite or standard"
+  default     = "standard"
+  validation {
+    condition     = contains(["lite", "standard"], var.plan)
+    error_message = "The specified plan is not a valid selection! Supported plans are: lite or standard"
+  }
+}
+
+variable "tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event notification will be created, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney)"
+  default     = "us-south"
+  validation {
+    condition     = contains(["us-south", "eu-gb", "eu-de", "au-syd"], var.region)
+    error_message = "The specified region is not valid, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney)"
+  }
+}
+
+variable "service_endpoints" {
+  type        = string
+  description = "Specify whether you want to enable the public, private, or both service endpoints. Supported values are 'public', 'private', or 'public-and-private'."
+  default     = "private"
+  validation {
+    condition     = contains(["public", "public-and-private", "private"], var.service_endpoints)
+    error_message = "The specified service endpoint is not a valid selection! Supported options are: public, public-and-private or private."
+  }
+}
+
+variable "cbr_rules" {
+  type = list(object({
+    description = string
+    account_id  = string
+    rule_contexts = list(object({
+      attributes = optional(list(object({
+        name  = string
+        value = string
+    }))) }))
+    enforcement_mode = string
+  }))
+  description = "(Optional, list) List of CBR rules to create"
+  default     = []
+}
+
+variable "skip_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to read the encryption key from the KMS instance. If set to false, pass in a value for the KMS instance in the existing_kms_instance_guid variable. In addition, no policy is created if var.kms_encryption_enabled is set to false."
+  default     = false
+}
+
+variable "kms_encryption_enabled" {
+  type        = bool
+  description = "Set this to true to control the encryption keys used to encrypt the data that you store in IBM Cloud® Databases. If set to false, the data is encrypted by using randomly generated keys. For more info on Key Protect integration, see https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect. For more info on HPCS integration, see https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs"
```

Comment:
This description is wrong. Its for ICD services. Needs to be updated for Event Notification service

Reply:
Done.